tutorpicasso/commands/enable_themes.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+from typing import Any
+
+import click
+from tutor import config as tutor_config
+
+
+@click.command(name="enable-themes", help="Enable picasso themes")
+def enable_themes() -> None:
+    """
+    Enable picasso themes.
+
+    This function enables the themes specified in the `PICASSO_THEMES` configuration
+    and applies them using the ThemeEnabler and ThemeGitRepository classes.
+    """
+    context = click.get_current_context().obj
+    tutor_root = context.root
+    tutor_conf = tutor_config.load(tutor_root)
+
+    if not tutor_conf.get("PICASSO_THEMES"):
+        return
+
+    for theme in tutor_conf["PICASSO_THEMES"]:  # type: ignore
```

Comment:
Why is `# type: ignore` needed in this file?

Reply:
Because we are using mypy in the test (from cookiecutter) and raise an error related to the tutor_config object type of data, saying that tutor_config["PICASSO_THEMES"] is not iterable.

Reply:
But can we fix it instead of ignoring it?

Reply:
The error I get is:
```bash
tutorpicasso/commands/enable_themes.py:24: error: Item "float" of "str | float | bool | list[str] | list[Any] | dict[str, Any] | dict[Any, Any] | None" has no attribute "__iter__" (not iterable)  [union-attr]
tutorpicasso/commands/enable_themes.py:24: error: Item "None" of "str | float | bool | list[str] | list[Any] | dict[str, Any] | dict[Any, Any] | None" has no attribute "__iter__" (not iterable)  [union-attr]
tutorpicasso/commands/enable_themes.py:24: error: Item "bool" of "str | float | bool | list[str] | list[Any] | dict[str, Any] | dict[Any, Any] | None" has no attribute "__iter__" (not iterable)  [union-attr]
```
It is because the object tutor_config that comes from Tutor has this union of types: https://github.com/overhangio/tutor/blob/eccb4d182ed62427f581085ac2a7877a9232955d/tutor/types.py#L12

I will try to fix it.

Reply:
Thank you for working on it, @MaferMazu! In any case, consider this comment non-blocking.

Reply:
Yes. I tried to improve the type, but it always returned another error about types.

Also, I think we shouldn't alter or handle that type error since the object comes from Tutor.

I'll add an inline comment and still ignore that type error.

Reply:
I already added the doc!